src/browser/context-menu-template.js

```diff
@@ -0,0 +1,99 @@
+var reload = function(item, focusedWindow) {
+  if (focusedWindow) {
+    focusedWindow.webContents.reload();
+  }
+};
+
+var windows_linux_template = [
+  {
+    label: 'Undo',
+    accelerator: 'CmdOrCtrl+Z',
+    role: 'undo:'
+  },
+  {
+    label: 'Redo',
+    accelerator: 'Shift+CmdOrCtrl+Z',
+    role: 'redo:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    role: 'cut:'
+  },
+  {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
```

Comment:
The difference is that OSx uses the `selector` property and non-OSx uses the `role` property on the menu item.